README.md

```diff
@@ -1,40 +1,120 @@
-# create-svelte
+# ForgeRock Web Login Framework
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
+## Warning: Vaporware
 
-## Creating a project
+**This is a prototype of a development framework for integrating a ForgeRock Login Widget into an existing app, or running as a standalone Login App. This project is not officially supported and is not recommended for any project development. If you use this, you accept all the risks that come with unsupported software.**
 
-If you're seeing this, you've probably already done this step. Congrats!
+## Technical Requirements
 
-```bash
-# create a new project in the current directory
-npm init svelte@next
+1. Node.js v16 or higher
+2. npm v8 or higher
```

Comment:
Lets add an `engines` property to package.json enforcing this, or a `.nvmrc` file?

Reply:
Added to the `engines` property. Thanks!